wgengine/userspace.go

```diff
@@ -1277,6 +1278,21 @@ func (e *userspaceEngine) SetLinkChangeCallback(cb func(major bool, newState *in
 	}
 }
 
+func (e *userspaceEngine) AddNetworkMapCallback(cb NetworkMapCallback) func() {
+	e.mu.Lock()
+	defer e.mu.Unlock()
+	if e.networkMapCallbacks == nil {
+		e.networkMapCallbacks = make(map[*networkMapCallbackHandle]NetworkMapCallback)
+	}
+	handle := new(networkMapCallbackHandle)
```

Comment:
`h` is fine instead of `handle` to reduce stutter. When a variable's scope is only a few lines, short is fine (and often preferable if it helps with stutter and thus readability)